_gsocprojects/2020/project_MCnet.md

```diff
@@ -0,0 +1,14 @@
+---
+project: MCnet
+layout: default
+logo: mcnet-logo.png
+description: |
+  [MCnet](http://montecarlonet.org) is a collaboration of the major collider event
```

Comment:
You should write the paragraph contiguous, otherwise words get connected to each other in the generated text.